core/src/main/scala/org/bitcoins/core/wallet/utxo/ExternalAddressTag.scala

```diff
@@ -0,0 +1,13 @@
+package org.bitcoins.core.wallet.utxo
+
+/**
+  * Address Tag Types defined outside the library, used for other projects
+  * creating there own address tags that aren't supported by bitcoin-s
+  */
+trait ExternalAddressTagType extends AddressTagType
+
+/**
+  * Address Tags defined outside the library, used for other projects
+  * creating there own address tags that aren't supported by bitcoin-s
```

Comment:
nit: "aren't natively supported" as we do support them using this trait :) (same above)